ExampleGenerator/Series/BarSeriesExamples.cs

```diff
@@ -1,4 +1,8 @@
-﻿namespace ExampleGenerator
+﻿using System;
```

Comment:
using directives should go inside namespace http://stylecop.soyuz5.com/SA1200.html